packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -347,12 +357,18 @@ export class Runtime {
         this.store.accounts.forEach((account, addr) => {
           this.updateBalance(txnParam, account);
         });
+        if (txnParam.type === TransactionType.DeleteSSC) {
+          this.deleteApp(txnParam.fromAccount.addr, txnParam.appId);
+        }
       }
     } else {
       // for a single (stand alone) transaction
       this.store.accounts.forEach((account, addr) => {
         this.updateBalance(txnParams, account);
       });
+      if (txnParams.type === TransactionType.DeleteSSC) {
+        this.deleteApp(txnParams.fromAccount.addr, txnParams.appId);
+      }
```

Comment:
We have repetitive code in this function (which will probably increase when we add more txn types). I would suggest to:
+ either move the common code to a function and reuse the function. OR
+ treat the single txn as a group of 1 txn (`const tx = [txnParams];`), and just operate the updates on an array once.

Reply:
second one seems good